style.css

```
@@ -0,0 +1,204 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+:root {
+    view-transition-name: root;
+}
+
+html {
+    display: block;
+}
+
+.header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 9px 400px;
+    width: 1920px;
+    height: 70px;
+    color: #ffffff;
+    margin: 0 auto;
+    gap: 10px;
+    border-bottom: 1px;
+}
+
+.logo{
+    width: 153px;
+    height: 51px;
+    margin: 0 auto;
+    display: block;
+    cursor: pointer;
+}
+
+.login{
+    width: 128px;
+    height: 48px;
+    border-radius: 8px;
+
+    gap: 10px;
+    margin: 0 200;
+    display: block;
+}
+
+.headline-wrap{
+    display: flex;
+    justify-content: center;
+    align-items:center;
+    padding-top: 200px;
+    width: 1920px;
+    height: 540px;
+    margin: 0 auto;
+    background-color: #CFE5FF;
+    width: 100%;
+}
+
+.headline{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 1110px;
+    height: 340px;
+    gap: 7px;
+    margin: 0 auto;
+}
+
+.headline-content{
+    display: flex;
+    position: relative;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 357px;
+    height: 260px;
+    padding: auto 60px;
+    gap: 32px;
+}
+
+.headline-heading{
+
+    margin: 0 auto;
+    height: 112px;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 56px;
+    font-family: Pretendard;
```

❗️ 수정요청
font-famliy 속성은 해당 요소들에 모두 공통 적용되므로 body 태그나 html 태그자체에 적용해주시고 나머지는 지우시는게 
반복을 줄이고 관리하기도 좋습니다.

```suggestion
html {
    font-family: Pretendard; // html 요소의 자식들은 해당 속성을 물려받음
}
```